configuration_go/k8sutil/k8sutil.go

```diff
@@ -0,0 +1,93 @@
+package k8sutil
+
+import (
+	"fmt"
+
+	mon "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring"
+	monv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
```

Comment:
nice idea, to simplify exact k8s resources.

Long term, perhaps even those utils have to be versioned by path, so you could migrate and host resources for different API version in the same config.